### ISSUE TITLE
Is This A "Free" Fork?

### ISSUE THREAD

**Issue by opticyclic** [DiffKit/DiffKit#5]
According to this message, the author took DiffKit commercial
 https://groups.google.com/forum/#!topic/diffkit-user/I_BBNOSbQOA

Is that statement true and is this an Apache Licensed fork from the last free version?

**Comment by jpanico** [comment]
No, that's not true. I have not worked on Diff'ng software since the
original DiffKit releases, and nobody has created a commercial fork of
DiffKit.

Joe
